CHANGES

```diff
@@ -1498,7 +1498,7 @@ www.mcs.anl.gov/mpi/mpich2/mpich2_1_0_6changes.htm.
   output.  Otherwise, the output message will be base on the error message
   associated with the mpi_errno parameter.
 
-- MPID_Segment_init() now takes an additional boolean parameter that specifies
+- MPIR_Segment_init() now takes an additional boolean parameter that specifies
```

Comment:
I'll add it in a fixup after the tests complete. I also left off one commit from your original branch. Trying to dig it up now.